VideoLocker/src/main/java/org/edx/mobile/view/adapters/InfiniteScrollUtils.java

```diff
@@ -138,7 +138,7 @@ public void reset() {
             activeLoadId.incrementAndGet(); // To disregard any in-progress loads
             adapter.clear();
             hasMoreItems = true;
-            loading = true;
+            loading = false;
```

Comment:
This still has us redundantly check the `loading` and `hasMoreItems` values.

It would be more correct to instead change the `loadMore` call below to `onLoadMore`